themes/V3/5ePHB/snippets/tableOfContents.gen.js

```diff
@@ -28,31 +28,33 @@ const getTOC = (pages)=>{
 	};
 
 	const res = [];
-	_.each(pages, (page, pageNum)=>{
-		if(!page.includes("{{frontCover}}") && !page.includes("{{insideCover}}") && !page.includes("{{partCover}}") && !page.includes("{{backCover}}")) {
-			const lines = page.split('\n');
-			_.each(lines, (line)=>{
-				if(_.startsWith(line, '# ')){
-					const title = line.replace('# ', '');
-					add1(title, pageNum);
+	const iframe = document.getElementById('BrewRenderer');
+	const iframeDocument = iframe.contentDocument || iframe.contentWindow.document;
+	const headings = iframeDocument.querySelectorAll('h1, h2, h3');
+
+	_.each(headings, (heading)=>{
+		const onPage = parseInt(heading.closest('.page,.phb').id?.replace(/^p/, ''));
+		const ToCExclude = getComputedStyle(heading).getPropertyValue('--TOC');
+		if(ToCExclude != '"exclude"') {
```

Comment:
This line is testing the wrong value. The value of `ToCExclude` would be `exclude`, not `"exclude"`.

Remove the double quotes.